server/monitor/src/main/resources/org/apache/accumulo/monitor/templates/navbar.ftl

```diff
@@ -33,12 +33,50 @@
           <ul class="navbar-nav ms-auto mb-2 mb-lg-0">
             <li class="nav-item dropdown">
               <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
-                <span id="statusNotification" class="icon-dot normal"></span>&nbspServers
+                <span id="statusNotification" class="status-icon normal">
+                  <svg id="statusNotificationDot" class="bi bi-circle-fill" width="16" height="16">
+                    <use xlink:href="../../resources/external/bootstrap/fonts/icons.svg#dot"></use>
+                  </svg>
+                  <svg id="statusNotificationCone" style="display:none;" class="bi bi-cone-striped" width="16" height="16">
+                    <use xlink:href="../../resources/external/bootstrap/fonts/icons.svg#cone"></use>
+                  </svg>
+                </span>
+                &nbsp;Servers
               </a>
               <ul class="dropdown-menu">
-                <li><a class="dropdown-item" href="/manager"><span id="managerStatusNotification" class="icon-dot normal"></span>&nbsp;Manager&nbsp;Server&nbsp;</a></li>
-                <li><a class="dropdown-item" href="/tservers"><span id="serverStatusNotification" class="icon-dot normal"></span>&nbsp;Tablet&nbsp;Servers&nbsp;</a></li>
-                <li><a class="dropdown-item" href="/gc"><span id="gcStatusNotification" class="icon-dot normal"></span>&nbsp;Garbage&nbsp;collector&nbsp;</a></li>
+                <li>
+                  <a class="dropdown-item" href="/manager">
+                    <div id="managerStatusIcon" class="status-icon normal">
+                      <svg id="managerStatusDot" class="bi bi-circle-fill" width="16" height="16">
```

Comment:
I don't think you need to add in these svg elements. You should just be able to specify the icon by adding the icon class name to the span tag, like what we were doing before. Also, you shouldn't put `<div>` tags, which are block elements, inside the `<li>` tags which are inline elements.

Reply:
So the dot icon was being created in screen.css with the icon-dot class. Since the icon was just a circle it was easy to create that with just css but the new cone icon (or most other complex icons) cannot easily be created with just css. For this reason I added the svg for the cone icon and to make it possible to switch the color of the current icon (whether its the dot or cone) using the same code, I opted to make the dot icon an svg as well. It makes it a lot easier and cleaner to change their color now that both icons are the same (svg).

Reply:
I think you might be able to do pure CSS to insert an icon (unicode char, not image) https://developer.mozilla.org/en-US/docs/Learn/CSS/Howto/Generated_content

Reply:
That might work but the icon would need to be part of the font. We could use the bootstrap font and then use the Unicode for these icons if thats what you mean. Or maybe there are a few icons that are typically included in the font we are using but I am not sure how many there are or if any are close to the idea we are trying to convey like the cone does.

Reply:
I think some characters are so ubiquitous, we can reasonably expect them to be rendered on a modern browser, without requiring special fonts. Like maybe ‼, ⁂, or maybe overloading ℠ (service mark) for safe mode could work. Or maybe the enclosed alphabetic characters Ⓜ, Ⓣ, and Ⓖ could be used to represent the status of each server type separately.

I'm just throwing ideas out there. I'm not actually sure which would maximize utility and aesthetics without trying them with the different colors and seeing what works best.

I'm not a huge fan of the graphical cone, personally... although it's better than doing nothing. My preference would be for it to be a simple character glyph, so it can follow our color scheme with the font color, or be a simple monochrome 2-D graphic with a transparent background so we can color it to fit the traffic light color scheme, rather than a full color 3-D image.

Reply:
What about something like &#x26A0;, ‼ or ⁂ within the colored circle? I can try a few out and see how they look.